ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/FileReaderByText/FileReaderValidation.java

```diff
@@ -0,0 +1,22 @@
+package main.java.edu.usb.argos.ASTProcessor.FileReaderByText;
```

Comment:
Move your validations class to a validations folder, the same with your other classes, divide it into folders to keep it more organized.

Reply:
Done